tests/m2m_through_regress/models.py

```diff
@@ -94,3 +94,31 @@ class CarDriver(models.Model):
 
     def __str__(self):
         return "pk=%s car=%s driver=%s" % (str(self.pk), self.car, self.driver)
+
+
+# Through models using multi-table inheritance
+class Event(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    people = models.ManyToManyField('Person', through='IndividualCompetitor')
+    special_people = models.ManyToManyField('Person', through='ProxiedIndividualCompetitor',
+                                            related_name='special_event_set')
+    teams = models.ManyToManyField('Group', through='CompetingTeam')
+
+
+class Competitor(models.Model):
+    event = models.ForeignKey('Event', models.CASCADE)
+    rank = models.IntegerField(blank=True, null=True)
+    disqualified = models.BooleanField()
```

Comment:
It's not clear to me if some of the extra fields like `disqualified` are needed. In general, it's nice to keep things minimal.